hw_5/task5.py

```diff
@@ -0,0 +1,256 @@
+import datetime
+import os
+import sys
+from hw_4.copy_module_4_file import lower_text, text_split_by_sentence, capitalize_sentences
+from task_for_module_7 import write_count_words_to_csv, write_letters_to_csv
+import sqlite3
+
+
+class SaveDataToDB:
+    def __init__(self, connection, cursor, text):
+        self.connection = connection
+        self.cursor = cursor
+        self.text = text
+
+    def publish_news_to_db(self, city):
+        self.connection.execute("INSERT INTO news (City, Text_field, Date_of_publication) "
+                                "VALUES (?, ?, ?)", (city, self.text, datetime.date.today().strftime('%Y-%m-%d')))
+        self.connection.commit()
+        self.cursor.execute('select * from news')
+        result = self.cursor.fetchall()
+        print(result)
+        self.cursor.close()
+        self.connection.close()
+
+    def publish_add_to_db(self, expiration_date):
+        current_date = datetime.datetime.now()
+        future_date_reformat = datetime.datetime.strptime(expiration_date, '%Y-%m-%d')
+        days_d = future_date_reformat - current_date
+        remaining_days = days_d.days
+        self.connection.execute("INSERT INTO private_add (Text_field, Expiration_date, Remaining_days)"
+                                "VALUES (?, ?, ?)", (self.text, expiration_date, remaining_days))
+        self.connection.commit()
+        self.cursor.execute('select * from private_add')
+        result = self.cursor.fetchall()
+        print(result)
+        self.cursor.close()
+        self.connection.close()
+
+    def publish_event_to_db(self, ticket_price, event_date):
+        self.connection.execute("INSERT INTO events (Text_field, Ticket_price, Event_date)"
+                                "VALUES (?, ?, ?)", (self.text, ticket_price, event_date))
+        self.connection.commit()
+        self.cursor.execute('select * from events')
+        result = self.cursor.fetchall()
+        print(result)
+        self.cursor.close()
+        self.connection.close()
+
+
+class NewsTool:
+    def __init__(self, user_input):
+        self.user_input = user_input
+
+    def start_program(self):
+        if self.user_input == '1':
+            text = input("Enter text: ")
+            city = input("Enter city: ")
+            b = News(text, city)
+            b.print_to_file()
+            connection = sqlite3.connect('publications.db')
+            cursor = connection.cursor()
+            cursor.execute('CREATE TABLE IF NOT EXISTS news (City text, Text_field text, Date_of_publication date)')
```

Comment:
Better create separate function, that creates all needed table before inserting data